config/newrelic.yml

```diff
@@ -1,34 +1,21 @@
-common: &default_settings
-  app_name: "identity-dashboard"
+production:
+  agent_enabled: true
+  app_name: pivcac.<%= Identity::Hostdata.env %>.<%= Identity::Hostdata.domain %>
```

Comment:
this is the dashboard repo... but this says `pivcac`?

Reply:
Got busted copy/pasting. How embarrassing. Willfix

Reply:
Fix: https://github.com/18F/identity-dashboard/pull/400/files